help/en/html/hardware/loconet/DigitraxMeters.shtml

```diff
@@ -31,41 +31,61 @@
       <h1><a id="Introduction" name="Introduction">
       Introduction to JMRI Volt and Current Meters with Digitrax Hardware</a></h1>
 
-      <p>Under certain configurations, JMRI provides tools under the "Tools" menu
-          item which may display "Track Voltage" and "Track Current" tools.  When
-          a LocoNet connection is defined and that connection is defined to use
-          any of certain Digitrax command stations, JMRI provides those tools.  When
-          a LocoNet connection is defined without a command station, or with 
-          any of the un-supported command stations, those tools will not be 
-          available. Those tools may also be available when a non-LocoNet connection
-          is defined and that connection type supports reporting of voltage and/or
-          current.</p>
-      <p>Note that "Track Current" is not necessarily a correct description of the
-          information that Digitrax command stations can provide.  See <a href="#Limitations_Current">Limitations</a>,
+      <p>Under certain configurations, JMRI provides a "Voltage/Current Meter" tool 
+          under the "Tools" menu item.  This tool may be used to display information
+          reported by certain Digitrax hardware.</p>
+      <p>The Voltage/Current Meter tool can report voltage or current data from
+          any LocoNet device which reports voltage and current information using
+          special Loconet messages.  In order to help differentiate the possible
+          sources, each source's data is identified by a name which includes the
+          device type and the device serial number.  Each is individually selectable
+          in the Voltage/Current Meter tool.</p>
+      <p>It is helpful to re-configure the Voltage/Current Meter tool's "Settings"
+          for displaying "2 integer digits" and "1 decimal digit".  This matches
+          the tool's display to the precision of the data reported on LocoNet.</p>
```

Comment:
@devel-bobm 
One simple solution for this is to add this to the Meter interface:

```
/**
 * Returns the number of integer digits this meter recommends for the display.
 *
 * @returns the number of integer digits, or 0 if no recommendation is given
 */
public default int getRecommendedIntegerDigits() {
    return 0;
}

/**
 * Returns the number of decimal digits this meter recommends for the display.
 *
 * @returns the number of decimal digits, or 0 if no recommendation is given
 */
public default int getRecommendedDecimalDigits() {
    return 0;
}
```

The MeterFrame would when check `Meter.getRecommendedIntegerDigits()` to check if it's value is non zero and if so, use it. The LnPredefinedMeters would then override these methods to return the desired number of digits. This would not break any of the other meters so it would be safe to add.

Reply:
New issue #9055 opened to track this feature request.